scripts/dependencies.R

```diff
@@ -0,0 +1,7 @@
+# Catch dependencies that renv might miss
```

Comment:
I like `components` for this better personally.

Reply:
I like it better too, to be honest.